eagle-xl/auto_plotter/gas_mass_functions.yml

```diff
@@ -39,3 +39,39 @@ adaptive_gas_HI_mass_function:
   observational_data:
     - filename: GalaxyHIMassFunction/Zwaan2003.hdf5
     - filename: GalaxyHIMassFunction/Haynes2011.hdf5
+
+gas_H2_mass_function:
+  type: "massfunction"
+  legend_loc: "lower left"
+  number_of_bins: 25
+  x:
+    quantity: "derived_quantities.gas_H2_mass_Msun"
+    units: Solar_Mass
+    start: 1e6
+    end: 1e11
+  y:
+    units: 1/Mpc**3
+    start: 1e-6
+    end: 0.316 # (1e-0.5)
+  metadata:
+    title: H_2 Mass Function
```

Comment:
You may as well use LaTeX formatting for H$_2$.

Reply:
Done.